src/main/java/org/verdictdb/jdbc41/VerdictStreamResultSet.java

```diff
@@ -0,0 +1,1329 @@
+    //
+    //        if (lastValue == null) {
+    //          return 0;
+    //        }
+    //
+    //        return (byte) TypeCasting.toByte(lastValue);
+    //      }
+    //      else {
+    //        throw new VerdictDBTypeException(queryResult.getValue(columnIndex-1));
+    //      }
+    //    }
+    //    catch (VerdictDBTypeException e) {
+    //      throw new SQLException(e.getMessage());
+    //    }
+  }
+
+  @Override
+  public byte getByte(String columnLabel) throws SQLException {
+    if (colNameIdx.containsKey(standardizedLabel(columnLabel))) {
+      return queryResult.getByte(standardizedLabel(columnLabel.toLowerCase()));
+    } else throw new SQLException("ColumnLabel does not exist.");
+  }
+
+  @Override
+  public byte[] getBytes(int columnIndex) throws SQLException {
+    checkIndex(columnIndex);
+    return queryResult.getBytes(columnIndex - 2);
+  }
+
+  @Override
+  public byte[] getBytes(String columnLabel) throws SQLException {
+    if (colNameIdx.containsKey(standardizedLabel(columnLabel))) {
+      return queryResult.getBytes(standardizedLabel(columnLabel.toLowerCase()));
+    } else throw new SQLException("ColumnLabel does not exist.");
+  }
+
+  @Override
+  public Reader getCharacterStream(int columnIndex) throws SQLException {
+    checkIndex(columnIndex);
+
+    throw new SQLFeatureNotSupportedException();
+  }
+
+  @Override
+  public Reader getCharacterStream(String columnLabel) throws SQLException {
+    throw new SQLFeatureNotSupportedException();
+  }
+
+  @Override
+  public Clob getClob(int columnIndex) throws SQLException {
+    checkIndex(columnIndex);
+
+    return queryResult.getClob(columnIndex - 2);
+  }
+
+  @Override
+  public Clob getClob(String columnLabel) throws SQLException {
+    if (colNameIdx.containsKey(standardizedLabel(columnLabel))) {
+      return queryResult.getClob(standardizedLabel(columnLabel.toLowerCase()));
+    } else throw new SQLException("ColumnLabel does not exist.");
+  }
+
+  @Override
+  public int getConcurrency() throws SQLException {
+    throw new SQLFeatureNotSupportedException();
+  }
+
+  @Override
+  public String getCursorName() throws SQLException {
+    throw new SQLFeatureNotSupportedException();
+  }
+
+  @Override
+  public Date getDate(int columnIndex) throws SQLException {
+    checkIndex(columnIndex);
+    return queryResult.getDate(columnIndex - 2);
+  }
+
+  @Override
+  public Date getDate(int columnIndex, Calendar cal) throws SQLException {
+    checkIndex(columnIndex);
+
+    throw new SQLFeatureNotSupportedException();
+  }
+
+  @Override
+  public Date getDate(String columnLabel) throws SQLException {
+    if (colNameIdx.containsKey(standardizedLabel(columnLabel))) {
+      return queryResult.getDate(standardizedLabel(columnLabel.toLowerCase()));
+    } else throw new SQLException("ColumnLabel does not exist.");
+  }
+
+  @Override
+  public Date getDate(String columnLabel, Calendar cal) throws SQLException {
```

Comment:
Can we support this method? I think people will complain in the same way for getTimestamp().

Reply:
Yes. I will add it.